scripts/utils.js

```diff
@@ -0,0 +1,8 @@
+Utils = {
```

Comment:
There is already a `Util ` in scripts/apps.js. Should be able to merge the two pretty trivially.